### PR TITLE
Use strong_parameters to require params

### PR DESCRIPTION
**Pull request by arthurnn**
We can handle require parameters now with strong parameters.

review @dwradcliffe

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -18,6 +18,9 @@ def set_locale
       # { :locale => I18n.locale }
   # end
 
+  rescue_from(ActionController::ParameterMissing) do |e|
+    render text: "Request is missing param '#{e.param}'", status: :bad_request
+  end
```
Comment (simi) [inline app/controllers/application_controller.rb]: *nitpick: Maybe one blank line will be good here.*

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -18,6 +18,9 @@ def set_locale
       # { :locale => I18n.locale }
   # end
 
+  rescue_from(ActionController::ParameterMissing) do |e|
+    render text: "Request is missing param '#{e.param}'", status: :bad_request
```
Comment (simi) [inline app/controllers/application_controller.rb]: What about to use one `rescue_from` for API and return text in requested format (JSON, YAML) and one here rendering just test?

Also is there any reason why text is not part of locales?

Comment (arthurnn) [inline app/controllers/application_controller.rb]: I was wondering to rescue differently for API vs web, but wanted to keep this simpler, and change afterwards if necessary.
The current behaviour right now is a render of text, even on API, so.

**Comment by dwradcliffe** [comment]
LGTM

**Pull request merged**